src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ManageWorkExperienceReferenceList.vue

```diff
@@ -96,10 +114,16 @@ export default defineComponent({
     const route = useRoute();
 
     const applicationStatus = (await getApplicationStatus(route.params.applicationId.toString()))?.data;
+    const certifications = (await getCertifications()).data!;
```

Comment:
can we use the `latestCertification `from `store/certification.ts`? Or do you need refresh here?

Reply:
Same question for `applicationStatus`.

Reply:
took latestCertification from store/certification. keping applicationStatus as an API call for now.